packages/compute-node.packages

```diff
@@ -1,5 +1,5 @@
-craycli=0.41.8-1
+craycli=0.41.11
 cray-switchboard=1.2.3-1
 cray-uai-util=1.0.13-1
 cfs-state-reporter=1.7.50-1
-cfs-trust=1.3.94-1
```

Comment:
What's this?

Reply:
Oh, that looks like maybe a trailing newline or something.  There is no real diff there. Nevermind.